main.go

```diff
@@ -16,6 +16,7 @@ var (
 	pgUser        = flag.String("user", "postgres", "Postgres username")
 	pgPass        = flag.String("pass", "", "Postgres password")
 	pgDB          = flag.String("db", "", "Postgres database")
+	pgNoSSL       = flag.Bool("no-ssl", false, "Disable ssl")
```

Comment:
Gflags (in C++) produces `--noX` flags for any boolean `--X` flag automatically. Unfortunately, `flag` doesn't do something like this. I'm a bit conflicted about `no-X` vs `X` with a default of true... WDYT?

Reply:
Below there's `showRelations` - a boolean flag with a default of true. So it should at least be consistent.

Reply:
How does a flag with a default of true even work? Does it support `-show-relations=false`?

Reply:
Because bool flags can be used just like this `-no-ssl` and this is pretty clear I think. It also is a matter of sane defaults, right? Turning ssl off should be explicit.

Reply:
Yup, not ideal. :)

Reply:
So I'll just merge it like that and then maybe switch to the flags package later on ...